platform/linux-generic/odp_queue_lf.c

```diff
@@ -355,11 +268,14 @@ void _odp_queue_lf_term_global(void)
 static void init_queue(queue_lf_t *queue_lf)
 {
 	int i;
+	odp_u128_t u128_t;
```

Comment:
`_t` postfix is usually used for data types, so this variable naming is a bit confusing.

Reply:
I will change to a more suitable name. Thanks!